mleap-spark-testkit/src/main/scala/org/apache/spark/ml/parity/SparkParityBase.scala

```diff
@@ -18,6 +19,10 @@ import ml.combust.mleap.spark.SparkSupport._
 import ml.combust.mleap.runtime.transformer.Pipeline
 import resource._
 
+import org.apache.spark.sql.Row
+import org.apache.spark.ml.linalg.{Vector, Vectors}
+import org.apache.spark.ml.util.TestingUtils._
```

Comment:
explicit import `relTol`

Reply:
Cannot only import `relTol` there're other internal implicits classes need import.
The `TestingUtils` class only contains functionality about equality test with numerical error. So usually we should import `TestingUtils._`